application-load-balancer/lambda_function/secret_rotator.py

```diff
@@ -0,0 +1,625 @@
+import json
+import os
+import boto3
+import logging
+import requests
+import time
+#import uuid
+from typing import Tuple, Dict, Any, List, Optional
+from slack_service import SlackNotificationService
+from requests.exceptions import RequestException
+
+from botocore.exceptions import ClientError
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+AWSPENDING = "AWSPENDING"
```

Comment:
Can we define these contants centrally rather than creating them twice?

Reply:
They have been removed from the second location in the Lambda.